src/utils/assets.js

```diff
@@ -266,3 +275,33 @@ export const getAssetSymbolByAddress = (assets: Asset[], supportedAssets: Asset[
 
   return assetSymbol;
 };
+
+export const mapAssetToAssetData = ({
+  symbol: token,
+  address: contractAddress,
+  name,
+  decimals,
+  iconUrl,
+}: Asset): AssetData => ({
+  token,
+  contractAddress,
+  name,
+  decimals,
+  tokenType: TOKENS,
+  icon: iconUrl ? `${SDK_PROVIDER}/${iconUrl}?size=3` : '',
+});
+
+export const mapCollectibleToAssetData = ({
+  contractAddress,
+  name,
+  id,
+  icon,
+}: Collectible): AssetData => ({
+  token: '',
+  decimals: 0,
+  contractAddress,
+  name,
+  id: id.toString(),
+  icon: icon || '',
+  tokenType: COLLECTIBLES,
+});
```

Comment:
There's a few empty strings here... i'm sure you know how i feel about empty strings by now 😂 is it better that we use `null` or is there a reason?

Reply:
It's due types in flow, if we add null then it's not accepted by `string`, this can be changed to `?string` and `|| null` would work, but then this leads to more `flow` fixes all around the app.